tests/apm/mod.rs

```diff
@@ -13,7 +13,7 @@ fn timed_query(_client: Client, command_result: &CommandResult) {
     };
 
     // Sanity check
-    assert!(duration >= 1500000000);
+    assert!(duration >= 500000000);
```

Comment:
What was the purpose of changing this? When I run the tests locally now, the assertion is failing (although it seems to be happening after the test is finished, which is another issue).

Given that the `sleep(500)` statements were not changed, not were the number of documents that would be slept on (sorry, awkward phrasing), there doesn't seem to be any reason to have changed this.